content/python/concepts/sets/sets.md

```diff
@@ -0,0 +1,115 @@
+---
+Title: "Sets" 
+Subjects:
+  - "Computer Science"
+  - "Data Science"
+Tags: 
+  - "Sets"
+  - "Data Types"
+CatalogContent: 
+  - "learn-python-3"
+  - "paths/computer-science"
+---
+
+A set is an unordered collection of elements with any duplicates.
+
+Sets are especially useful for performing logical operations like finding the union, intersection, or difference between collections of elements. For example, sets could be used to determine mutual friends on a social networking site.
+
+## Creating a Set
+
+There are several ways to create a set. 
+
+1. We can create a set from a list with the `set()` keyword:
+
+```py
+animals = set()
+animals = set(['aardvark', 'snail', 'squid'])
+```
+
+2. A set can also be created with curly braces:
+
+```py 
+oceans = {'Pacific', 'Atlantic', 'Indian', 'Southern', 'Arctic'}
+```
+
+It's important to note that creating a set will remove any duplicate elements. For example:
+
+```py
+dog_breeds = {'pug', 'golden retriever', 'corgi', 'pug', 'corgi'}
+```
+
+will create a new set with only unique values:
+
+```py 
+{'pug', 'golden retriever', 'corgi'}
+```
+
+## Set Operations
+
+Sets are especially useful for logical operations such as finding the intersection, union, or difference between collections. 
+
+## Union
+
+The `union()` method or `|` operator can be used to merge two or more sets:
+
+```py
+liked_artists1 = {'Grimes', 'Billie Eilish'}
+liked_artists2 = set({'Daft Punk', 'Iggy Azalea'})
+
+all_likes = liked_artists1.union(liked_artists2)
+print(all_likes)
+```
+
+The output would be:
+
+```shell
+{'Iggy Azalea', 'Daft Punk', 'Grimes', 'Billie Eilish'}
+```
+
+## Intersection
+
+We can use the `intersection()` method or `&` operator to find the shared elements within two or more sets. For example:
```

Comment:
Add a period in `.intersection()` method.

Reply:
Thank you! I added a period to the front of the .intersection() method and the methods mentioned below.